packages/foundation/docs/themes.mdx

```diff
@@ -1,30 +1,232 @@
+    ▶︎ The tertiary color is like a sparkle here and there, not commonly used but meant to be very visible when is used.
+  positive: { ... },
+    ▶︎ Commonly used to show a reaction to an action that was successfully performed, e.g. A form submition.
+  negative: { ... },
+    ▶︎ Used to show a fallback action, meaning the less ideal action that we intend our users to do
+  error: { ... },
+    ▶︎ As its name says, this is used to show bright error messages, e.g. A form submittion that failed to be validated.
+}
+
+```
+
+You can see the categories enum definition here: [ColorCategories](https://github.com/inavac182/uireact/blob/main/packages/foundation/src/types/enums/color-categories.ts)
+
+### Tokens
+
+Each coloration should include 5 levels of the main color, these are referrer to as tokens:
+
+```
+{
+  token_10: '',
+    ▶︎ Lighter
+  token_50: '',
+  token_100: '',
+    ▶︎ Main color
+  token_150: '',
+  token_200: '',
+    ▶︎ Darker
+}
+```
+
+From lighter to darker, 10 → 200. The idea is that we keep some flexibility in case we need to add additional levels to our colorations
+but as of now we believe these levels should be enough to provide a good UX.
+
+We suggest you to start with the main color and then work your way outside to both edges of the spectrum.
+A good color spectrum would be one where if you render a text with level 200 on top of a background level 10 of the same category it should read just fine.
+
+Typing for tokens is defined [here](https://github.com/inavac182/uireact/blob/main/packages/foundation/src/types/themes/internal/tokens.ts#L3)
+
+### Light Coloration 🤍
+
+You can see the light coloration of the provided theme here: [Light Coloration](https://github.com/inavac182/uireact/blob/main/packages/foundation/src/themes/default-colors/light.ts)
 
 <ThemeTokens colors={DefaultTheme.light} />
 
-## Dark coloration
+### Dark Coloration 🖤
 
-[Dark Theme](https://github.com/inavac182/uireact/blob/main/packages/foundation/src/themes/default-colors/dark.ts)
+You can see the dark coloration of the provided theme here: [Dark Theme](https://github.com/inavac182/uireact/blob/main/packages/foundation/src/themes/default-colors/dark.ts)
 
 <ThemeTokens colors={DefaultTheme.dark} />
 
-## Structure
+## Sizes 🎢
 
-Each theme has to meet the structure of the [Theme](https://github.com/inavac182/uireact/blob/main/packages/foundation/src/types/themes/theme.ts) type.
+Currently we provide with the size set up for headings and normal texts.
+
+```
+{
+  headings: { ... },
+    ▶︎ Heading sizes
+  texts: { ... }
+    ▶︎ Text sizes
+}
+```
+
+Typing for [Sizes](https://github.com/inavac182/uireact/blob/main/packages/foundation/src/types/themes/internal/sizes.ts#L11) definition
+
+### Headings
+
+The headings follows the ARIA convention and you could set up to 6 different sizes for [Headings](https://github.com/inavac182/uireact/blob/main/packages/foundation/src/types/enums/heading-levels.ts), we suggest you only differentiate
```

Comment:
it may be worth to add a reference to the `ARIA convention` mentioned here, so I can understand why this decision was made